sdk/src/types/api/core.rs

```diff
@@ -10,11 +10,17 @@ use alloc::{
 
 use serde::{Deserialize, Serialize};
 
+// Re-export types that are also used as responses.
```

Comment:
Can't say I'm a big fan of this

Reply:
did make a bit more sense with my preferred option, but this allows us to import all the responses in the same import group.

Reply:
because that's not how it's defined in the TIP :trollface:

Reply:
type aliases would also work

Reply:
I would still definitely keep `SlotCommitment` as is, doesn't really make sense to use it under two names. If a type we already have can be used as a response, then so be it.

Reply:
Please remove the `Commitment` alias

Reply:
I think you want that for `BlockResponse` and `OutputMetadataRespone` as well, right?

Reply:
I am less opinionated about them but yeah I'd personally remove all these re-exports